library/types/test/hostname_test.rb

```diff
@@ -10,30 +10,49 @@
 
   let(:etc_hostname) { "etc.hostname.cz" }
   let(:cmd_hostname) { "cmd.hostname.cz" }
+  let(:hostname)     { Yast::Hostname }
+
+  def allow_execute_hostname(result)
+    ret = { "stdout" => result, "exit" => 0 } if result
```

Comment:
Actually this makes `ret` nil if `result` is nil, but I don't think ".target.bash_output" ever returns nil.
I can get
```rb
{
"exit"=>1, 
"stderr"=>"sh: -c: line 0: syntax error near unexpected token `newline'\n" \
               "sh: -c: line 0: `nonsensecommand('\n", 
"stdout"=>""
}
```

or
```rb
{
"exit"=>127, 
"stderr"=>"sh: nonsensecommand: command not found\n", 
"stdout"=>""
}
```

So I think it is better to specify the entire return value of `.target.bash_output`, not just the stdout part.